package.json

```diff
@@ -21,51 +21,31 @@
       "plugins": {
         "uiHierarchy": "enabled",
         "screenshot": {
-          "enabled": true,
-          "shouldTakeAutomaticSnapshots": true,
-          "keepOnlyFailedTestsArtifacts": true,
-          "takeWhen": {
-            "testStart": true,
-            "testDone": true,
-            "appNotReady": true
-          }
+          "enabled": false
```

Comment:
@Ginny91 I never use the screen shots and they are slow on circle CI. So I disabled them. However, I'm not sure this is the right choice. What do you think?